src/data/monoid_algebra.lean

```diff
@@ -50,7 +50,7 @@ The monoid algebra over a semiring `k` generated by the monoid `G`.
 It is the type of finite formal `k`-linear combinations of terms of `G`,
 endowed with the convolution product.
 -/
-@[derive [inhabited, add_comm_monoid]]
+@[derive [inhabited, add_comm_monoid, has_coe_to_fun]]
```

Comment:
I'm confused about what's going on here. Judging from lemmas like `mul_single_one_apply`, it already seems to be possible to apply values of `monoid_algebra` type as though they were functions.

Reply:
This discussion belongs in #4315